src/ripple/ledger/impl/ReadView.cpp

```diff
@@ -66,7 +62,7 @@ class Rules::Impl
     bool
     enabled (uint256 const& feature) const
     {
-        if (presets_ && presets_->count(feature) > 0)
+        if (presets_.count(feature) > 0)
```

Comment:
The two functions should be equivalent in terms of performance. I _think_ `count` is a standard idiom for deciding if a set or map contains an element (you see it in our code and in boost::program_options, for example). And `presets_.count(...)` is terser than `presets_.find(..) != presets_.end()`.

Having said all that, this is definitely a nit. I prefer `count`, but I'm happy to change it if you strongly prefer `find`.